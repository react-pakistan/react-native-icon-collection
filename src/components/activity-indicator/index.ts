export {
  ActivityIndicator,
  ActivityIndicatorSize,
} from './activity-indicator';
