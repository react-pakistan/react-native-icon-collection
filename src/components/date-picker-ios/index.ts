export { DatePickerIOS } from './date-picker-ios';
