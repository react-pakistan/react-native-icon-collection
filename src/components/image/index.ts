export {
  IImageProps,
  Image,
  ResizeMethod,
  ResizeMode,
} from './image';
