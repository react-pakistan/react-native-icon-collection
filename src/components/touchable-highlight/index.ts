export { TouchableHighlight } from './touchable-highlight';
