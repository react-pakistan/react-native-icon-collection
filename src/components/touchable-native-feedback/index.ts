export { TouchableNativeFeedback } from './touchable-native-feedback';
