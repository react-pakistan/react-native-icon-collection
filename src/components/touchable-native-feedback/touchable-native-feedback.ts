import styled from 'styled-components/native';

export const TouchableNativeFeedback = styled.TouchableNativeFeedback``;
