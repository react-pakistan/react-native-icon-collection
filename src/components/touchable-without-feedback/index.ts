export { TouchableWithoutFeedback } from './touchable-without-feedback';
