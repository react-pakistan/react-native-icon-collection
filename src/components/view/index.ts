export { View, IView } from './view';
