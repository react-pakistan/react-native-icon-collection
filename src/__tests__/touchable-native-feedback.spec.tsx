import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';
import { TouchableNativeFeedback } from '../components/touchable-native-feedback';
// import {
// } from '../story/view.story';

describe('Testing TouchableNativeFeedback component', () => {
  it('matches TouchableNativeFeedback snapshot', () => {
    const wrapper = renderer.create(
      <TouchableNativeFeedback>
        <Text>
          Hello World
        </Text>
      </TouchableNativeFeedback>,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
