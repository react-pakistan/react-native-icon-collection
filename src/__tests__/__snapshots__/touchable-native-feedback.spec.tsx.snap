// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing TouchableNativeFeedback component matches TouchableNativeFeedback snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffbcbc",
        "borderColor": "red",
        "borderWidth": 1,
        "height": 100,
        "justifyContent": "center",
        "margin": 10,
        "width": 300,
      },
      Array [
        Object {},
      ],
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#333333",
        "margin": 20,
      }
    }
  >
    TouchableNativeFeedback is not supported on this platform!
  </Text>
</View>
`;
